apps/calendar/js/app.js

```diff
@@ -229,9 +228,14 @@ module.exports = {
 
     // Handle aria-selected attribute for tabs.
     tablist.addEventListener('click', (event) => {
-      if (event.target !== today) {
-        AccessibilityHelper.setAriaSelected(event.target, tabs);
+      if (event.target === today) {
+        return;
       }
+
+      Array.prototype.forEach.call(
+        tabs,
+        tab => tab.setAttribute('aria-selected', 'false')
+      );
```

Comment:
[`[aria-selected]`](http://www.w3.org/TR/wai-aria/states_and_properties#aria-selected) is used by screen readers to specify which `tab` is active.

more info about the screen reader: https://wiki.mozilla.org/Accessibility/Mobile/ScreenReader